index.js

```diff
@@ -250,8 +250,8 @@ class Paper {
             // compile all the template required files into an object
             result = {};
             for (let path in templatePath) {
-                renderPromises.push(this.render(path, data.context).then(html => {
-                    result[path] = html;
+                renderPromises.push(this.render(templatePath[path], data.context).then(html => {
```

Comment:
This doesn't make sense for Arrays. The function signature says `@param  {String|Array} templatePath A single template or list of templates to render.` but this code implies that it is an object.

Reply:
When a for in loop is done on an array it uses numeric indices so templatePath[path] for the first element of an array is the same as templatePath[0].